src/sbt-test/sbt-release/tag-default/test

```diff
@@ -0,0 +1,11 @@
+$ exec git init .
+$ exec git add .
+$ exec git commit -m init
+$ exec git tag v0.1.0
+
+-> release with-defaults
+-> release with-defaults tag-default a
+
+> release with-defaults tag-default o
+> release with-defaults tag-default k
+> release with-defaults tag-default v0.1.0-debug
```

Comment:
I don't understand this scripted tests - why does `release with defaults` fail and `release with-defaults tag-default a` also fail, but `release with-defaults tag-default o` doesn't fail?  What's so special about `o` that makes it different to `a`?  I just don't get it.  Could you add some comments here to explain what's being tested and what's going on?